src/nvim/eval.c

```diff
@@ -1534,20 +1543,20 @@ ex_let_vars (
     EMSG(_("E688: More targets than List items"));
     return FAIL;
   }
-  // l may actually be NULL, but it should fail with E688 or even earlier if you
-  // try to do ":let [] = v:_null_list".
+  // lt may actually be NULL, but it should fail with E688 or even earlier if
```

Comment:
Is `lt` a typo for `It` or should it stay `l`?

Reply:
Oh, I thought that `l` there was a typo for `It`, but it appears I actually wrote variable name there.